content/applications/hr/recruitment/schedule_interviews.rst

```diff
@@ -0,0 +1,325 @@
+===================
+Schedule interviews
+===================
+
+Schedule in-person, virtual, and phone interviews in Odoo through the *Recruitment* app.
+
+An interview can be scheduled in one of two ways: either by the :ref:`recruitment team
+<schedule_interviews/recruitment-scheduled>` , or by the :ref:`applicant
+<schedule_interviews/applicant-scheduled>` themselves.
+
+.. _schedule_interviews/recruitment-scheduled:
+
+Recruitment team scheduled interviews
+=====================================
+
+When an applicant reaches the interview stage, the recruitment team should schedule the interview by
+first coordinating a suitable date and time with the applicant and interviewers.
+
+To schedule the interview, navigate to the applicant's card by going to the
+:menuselection:`Recruitment app`, and click the relevant job card. This opens the
+:guilabel:`Applications` page for that job position. Then, click the applicant's card to view the
+detailed applicant form.
+
+To schedule an interview, whether a phone, virtual, or in-person interview, click the :guilabel:`No
+Meeting` smart button at the top of the applicant's record.
+
+.. note::
+   The meeting smart button displays :guilabel:`No Meeting` if no meetings are currently scheduled.
+   For new applicants who are new to the :guilabel:`First Interview` stage, this is the default.
+
+   If there is one meeting already scheduled, the smart button displays :guilabel:`1 Meeting`, with
+   the date of the upcoming meeting beneath it. If more than one meeting is scheduled, the button
+   displays :guilabel:`Next Meeting`, with the date of the first upcoming meeting beneath it.
+
+Doing so loads the *Calendar* application, showing the scheduled meetings and events for the
```

Comment:
I see what you mean- I spoke with Sam about this too, it is probably the actual calendar app loading, but the masthead of Recruitment stays the same (I have seen this in other apps as well). Since it is confusing to users, or could be, I am just going to say "a calendar loads" since that is still correct.